### ISSUE TITLE
Spell check ignores user-defined environments

### ISSUE THREAD

**Issue by zavandr** [streetsidesoftware/vscode-spell-checker#4235]
The spell-checking does not work inside new LaTeX environments of the form

\myenv{ This text is not spell checked }

but is works elsewhere (see the attached screenshot).

How can I include my custom environments into the spell-checking?

P.S. My text editor is VSCode

![Image](https://github.com/user-attachments/assets/5453091c-e191-4eab-89a0-dbe9b4bde95a)

**Comment by Jason3S** [comment]
@zavandr,

Thank you for the good example.

The issue is that the spell checker tries to ignore random macros/commands because they create a lot of false positives.

The pattern used to ignore macros is defined here: [dictionaries/latex/cspell-ext.json#L29-L31](https://github.com/streetsidesoftware/cspell-dicts/blob/e677b387db0719f904b418a1fe819614f6af55e7/dictionaries/latex/cspell-ext.json#L29-L31)

There are two basic options:
1. Redefine the pattern in your own configuration (not easy).
2. Prefix you command with one of the following words:
`title`, `color`, `section`, `subsection`, `footnote`, `chapter`, `part`, `caption`, `emph`, `enquote`, `text`.


Example using accepted prefix:
```latex
\newcommand{\textmyenv}[1]{#1}

% ...
\textmyenv{Should be spell checked.}
```

**Comment by Jason3S** [comment]
@zavandr,

Did it work?

**Comment by zavandr** [comment]
Yes, I prefixed it with `text` and it worked. Thanks.
